src/log/log.c

```diff
@@ -1904,9 +1904,9 @@ __log_has_hole(WT_SESSION_IMPL *session, WT_FH *fh, wt_off_t log_size,
 			 * hole.  We can do some simple data verification to
 			 * ensure that this part of the file is not corrupted.
```

Comment:
I suggest getting rid of the "We can do..." sentence here. This code right here is only walking within the buffer to find the non-zero part. We're not verifying here yet.